frontend/tsconfig.json

```diff
@@ -8,7 +8,9 @@
         "noImplicitThis": true,
         "isolatedModules": true,
         "allowJs": true,
-        "moduleResolution": "node"
+        "moduleResolution": "node",
+        "resolveJsonModule": true,
+        "esModuleInterop": true
```

Comment:
Why was this added? To import json files in the code? Modifications to the build are almost always more appropriate in the webpack config since we are using webpack. If you need to import json (which shouldn't be necessary if the interests are stored on the backend), then a json-loader in the webpack config would be more appropriate.